torchx/cli/cmd_run.py

```diff
@@ -113,6 +113,23 @@ def run(self, args: argparse.Namespace) -> None:
             print(get_builtin_source(builtin_name))
 
 
+class SchedulerArgOnce(argparse.Action):
+    scheduler_args_set = False
```

Comment:
Do we need this?

Reply:
Yes, it's a one-shot which is set to true when this arg is parsed. This can be converted to a set containing the arg so the class can be generalized to work for more cases.